docs/source/developer/contributing.rst

```diff
@@ -415,12 +421,12 @@ It is built with `Sphinx <http://sphinx-doc.org/>`_ and
 `numpydoc <https://numpydoc.readthedocs.io/>`_, which are available using
 conda or pip; i.e. ``conda install sphinx numpydoc``.
 
-To build the documentation, you need the bootstrap theme::
+To build the documentation, you need the bootstrap and rtd themes::
```

Comment:
Is this still accurate? RTD doesn't seem to install `sphinx_bootstrap_theme`: e.g. https://readthedocs.org/api/v2/build/15944491.txt

Reply:
I don't think it needs `sphinx_bootstrap_theme`.